frontend/src/layout/navigation/TopBar/SitePopover.tsx

```diff
@@ -268,6 +295,7 @@ export function SitePopover(): JSX.Element {
                             {!preflight?.cloud && <License license={relevantLicense} expired={expired} />}
                             <SystemStatus />
                             {!preflight?.cloud && <Version />}
+                            <AsyncMigrations />
```

Comment:
FYI @tiina303, removed the check as we already check above that we don't show this section on cloud except for staff users